koala/ast/Range.py

```diff
@@ -0,0 +1,299 @@
+
+import re
+from collections import OrderedDict
+# from excelutils import col2num
+import string
+def col2num(col):
+    
```

Comment:
Yep, but for some reason i couldn't use it here (python system files issues), and i didn't to spend 4252325252 hours on solving it. So i just copied it in the file so i could continue working.